thrifty-kotlin-codegen/src/test/kotlin/com/microsoft/thrifty/kgen/KotlinCodeGeneratorTest.kt

```diff
@@ -357,7 +357,7 @@ class KotlinCodeGeneratorTest {
 
         file.single().toString() should contain("""
             |public interface Svc {
-            |  public suspend fun doSomething(foo: Int): Int
+            |  public abstract suspend fun doSomething(foo: Int): Int
```

Comment:
I'm so confused.  Why does a generated interface need to include the `abstract` keyword?  Isn't it implicit in the fact the the function belongs to an interface and has no body?

Reply:
Totally agree with you. KotlinPoet is a bit weird here. Although the redundant modifier is only cosmetic, I am curious and checking out the relevant code :)

Reply:
I see; I guess it may be another "library-mode" change.  That sure junked up our generated code.  Guess it's a net win though.

Reply:
The weird thing is not that it now emits the keyword when you explicitly specify it (that sounds very much correct).
The weird thing is that you can not NOT explicitly make a function in an interface abstract (which is why thrifty also needs to add an abstract marker).
The notion of implicit abstractness of a interface function does not exist.